example/lib/main.dart

```diff
@@ -48,38 +87,213 @@ class _MyAppState extends State<MyApp> {
     return MaterialApp(
       home: Scaffold(
         appBar: AppBar(
+          backgroundColor: Color.fromRGBO(
+            141,
+            8,
+            1,
+            2,
+          ),
```

Comment:
you can move this color (and the other ones below) to a constants file and make the widget more readable